docs/src/architecture/09_architecture_decisions/0010-frontend-json-schema-constraints.md

```diff
@@ -0,0 +1,170 @@
+---
+    title: 0010 Frontend JSON Schema Constraints
+    adr:
+        author: Dominik Toton
+        created: 14-Apr-2025
+        status:  draft
+    tags:
+        - flutter
+---
+
+## Context
+
+[JSON Schema Draft-07] is a standard chosen to describe templates for various documents with which the frontend app must work.
+The responsibilities of the frontend app are to understand the JSON Schema structure and build meaningful UI components from it.
+With the complexity of the JSON Schema it's important to understand the constraints that the frontend app has.
+
+## Assumptions
+
+* Some of the app logic needs are not encoded directly in the JSON Schema document templates
+ therefore the app adds it's own logic on top of the JSON Schema logic.
+* There is no immediate plan to update the JSON Schema templates to satisfy the needs of the app due to resource constraints.
```

Comment:
This is not correct.  JSON Schema defines the structure and validity of JSON Documents, it isn't intended to define application behavior or logic.  Its not a programming language.  Its not a case of "Resource Constraints".

How the UI handles the document and what parts of it, it thinks are important are UI concerns and not related to the document schema at all..